packages/service/src/ui-service/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const Endpoints = {
+    arm: `https://management.azure.com`,
+};
+
+export const ApiVersion = {
```

Comment:
We should be explicit about what API versions these are for, since they're different for every RP.